pkg/kubectl/cmd/auth/reconcile.go

```diff
@@ -29,21 +28,25 @@ import (
 	internalrbacclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/rbac/internalversion"
 	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/genericclioptions"
 	"k8s.io/kubernetes/pkg/kubectl/resource"
+	"k8s.io/kubernetes/pkg/printers"
 	"k8s.io/kubernetes/pkg/registry/rbac/reconciliation"
 )
 
 // ReconcileOptions is the start of the data required to perform the operation.  As new fields are added, add them here instead of
 // referencing the cmd.Flags()
 type ReconcileOptions struct {
+	PrintFlags      *genericclioptions.PrintFlags
+	FilenameOptions *resource.FilenameOptions
+
 	Visitor         resource.Visitor
 	RBACClient      internalrbacclient.RbacInterface
 	NamespaceClient internalcoreclient.NamespaceInterface
 
-	Print func(*resource.Info) error
+	PrintObject printers.ResourcePrinterFunc
```

Comment:
This interface here `printers.ResourcePrinterFunc` should have a corresponding one in `genericclioptions` for people to use.

Reply:
Done